### PR TITLE
Add search pages permission.

### PR DESCRIPTION
**Pull request by miri**
#112

### CODE REVIEW

**Comment by miri** [comment]
image

File: haskala/modules/custom/haskala_general/haskala_general.features.user_permission.inc [hunk 1]
```diff
@@ -10,24 +10,24 @@
 function haskala_general_user_default_permissions() {
   $permissions = array();
 
-  // Exported permission: 'search content'.
-  $permissions['search content'] = array(
-    'name' => 'search content',
+  // Exported permission: 'access haskala search_api_page'.
```
Comment (amitaibu) [inline haskala/modules/custom/haskala_general/haskala_general.features.user_permission.inc]: permissions shouldn't be exported by features, it should be done via the ``.profile``, so an admin may change them without "overriden features" appearing. Example:

https://github.com/Gizra/negawatt-server/blob/master/negawatt/negawatt.profile#L28-L31
https://github.com/Gizra/negawatt-server/blob/master/negawatt/negawatt.profile#L72-L91

**Comment by miri** [comment]
image

**Comment by bitamar** [comment]
@miri  - Please "merge squash" into master:
```
git checkout master
git pull
git merge --squash 112
git commit -am "#112: Adding search page permissions"
git push origin master
```

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**